src/System.Net.Security/System.Net.Security.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
```

Comment:
I will fix this, its because I had to add files to the solution so the version number came along for the free ride :)